src/core/symbols/line_symbol.cpp

```diff
@@ -803,23 +803,19 @@ void LineSymbol::processContinuousLine(
 	{
 		auto mid_position = (split.clen + effective_end_length - mid_symbols_length) / 2;
 		auto next_split = SplitPathCoord::at(mid_position, split);
-		path.copy(split, next_split, processed_flags, processed_coords);
-		split = next_split;
 		
 		auto orientation = qreal(0);
-		bool mid_symbol_rotatable = bool(mid_symbol) && mid_symbol->isRotatable();
```

Comment:
Removal of the test for `nullptr` makes me ask how come the `mid_symbol` pointer won't be NULL?

Reply:
I removed that check since it's already done some lines above: `set_mid_symbols = set_mid_symbols && mid_symbol && ...`

Reply:
Ah, yes. I overlooked that test. Thanks for the clarification and sorry for the noise.